### DISCUSSION TITLE
Documentation for IDistributedLockEngine

### DISCUSSION THREAD

**Discussion by natilivni** [dotnet/dotNext#35] [Q&A] [answered]
I see in the source code that you have already built a distributed lock engine.  However, I do not see documentation on how to setup/use this feature.  Can you please point me in the right direction?  (this is the exact use case I am trying to build the cluster for).  Further, does your pattern guarantee fair access (ie.  The critical section of code will be accessed in the same order in which the locks were acquired)

**Comment by sakno** [comment, marked as answer]
`IDistributedLockEngine` was removed and never published as a part of the official release.
